src/main/java/org/apache/flink/state/benchmark/StateBenchmarkBase.java

```diff
@@ -20,15 +20,17 @@
 import org.apache.flink.benchmark.BenchmarkBase;
 import org.apache.flink.contrib.streaming.state.benchmark.StateBackendBenchmarkUtils;
 import org.apache.flink.runtime.state.KeyedStateBackend;
-
```

Comment:
Are you using the same formatting settings as for the main Flink repository? 

https://nightlies.apache.org/flink/flink-docs-master/docs/flinkdev/ide_setup/#code-formatting

Reply:
updated